src/client/components/pages/parts/editor-profile.js

```diff
@@ -76,7 +76,7 @@ class EditorProfileTab extends React.Component {
 								href="/editor/edit"
 								title="Edit basic editor info"
 							>
-								<FontAwesome name="pencil"/>{' '}Edit Profile
+								<FontAwesomeIcon icon="pencil"/>{' '}Edit Profile
```

Comment:
pencil -> pencil-alt
It's already in the icon library, just needs to be renamed